components/NavBar.js

```diff
@@ -1,12 +1,14 @@
+import { useAuth0 } from '@auth0/auth0-react'
 import { useRouter } from 'next/router'
-import { useAuth } from 'use-auth0-hooks'
 
 import NavLink from './NavLink'
 
 export default function NavBar (props) {
   const { pathname } = useRouter()
-  const { isAuthenticated, isLoading } = useAuth()
+  const { isAuthenticated, isLoading } = useAuth0()
+  // FIXME: Fetch adminUser?
```

Comment:
This FIXME needs a bit more context.

Reply:
Fixed in https://github.com/ibi-group/otp-admin-ui/pull/30/commits/74071aafc6203b31aff0c8e777868107671e4506. Good catch. This was an experiment that went nowhere.